nic_operator/nic_operator_ci_test.sh

```diff
@@ -300,6 +305,8 @@ function configure_common {
 function configure_ofed {
     local file_name="$1"
 
+    modprobe -r rpcrdma
```

Comment:
why this is need?

Reply:
following the bug https://redmine.mellanox.com/issues/2248725, the rdma-core package installs a udev rule that would actually load the rpcrdma module into the system at reboot, and since the module is not managed by the ofed, the ofed would fail to unload it. This line is a precaution to the scenario that the rpcrdma module was not blacklisted or was loaded on the system by the rdma-core for any other reason.